app/lib/frontend/handlers/account.dart

```diff
@@ -433,6 +443,7 @@ Future<shelf.Response?> checkAuthenticatedPageRequest(
     );
     final signInUri = Uri(path: '/sign-in', queryParameters: {
       'go': goUri.toString(),
+      if (includeWebmasterScope) 'webmaster': '1',
```

Comment:
Why not make this more generic?

Like `scopes=`, then we can just have a constant `allowedScopes` and if `scopes` passed by URL is not a subset of `allowedScopes` then we return 400.

Reply:
Slightly nicer/shorter URLs this way.

Reply:
But we a URL that is more confusing, and we hardcode logic related to publishers into the login-flow (we probably can't avoid that), but we could keep it out of the public interface.

If one day we need to request different scopes somewhere else, `&scopes=` checked against a hardcoded allowlist of scopes is very easy to extend.

Anyways, just a suggestion.